wuzz.go

```diff
@@ -832,7 +818,7 @@ func setViewTextAndCursor(v *gocui.View, s string) {
 	v.SetCursor(len(s), 0)
 }
 
-func quit(g *gocui.Gui, v *gocui.View) error {
+func quit(_ *gocui.Gui, _ *gocui.View) error {
```

Comment:
I suppose this is the proper go way, but `go fmt` does not correct it.
This mod is OK to me in another PR.